File: beecrowd/1008.cpp
// created by mostafizzfuad
#include <bits/stdc++.h>
using namespace std;
int main() {

    int number, workingHours;
    double amountPerHour;
    cin >> number >> workingHours >> amountPerHour;

    double salary = workingHours * amountPerHour;

    cout << "NUMBER = " << number << endl;
    cout << "SALARY = U$ " << fixed << setprecision(2) << salary << endl;
    
}

File: beecrowd/1008.cs
// created by mostafizzfuad
using System;
class URI
{
    static void Main(string[] args)
    {

        int number = Convert.ToInt32(Console.ReadLine());
        int workingHours = Convert.ToInt32(Console.ReadLine());
        double amountPerHour = Convert.ToDouble(Console.ReadLine());

        double salary = workingHours * amountPerHour;

        Console.WriteLine("NUMBER = " + number);
        Console.WriteLine("SALARY = U$ " + salary.ToString("0.00"));

    }
}